modules/core/04-channel/keeper/upgrade.go

```diff
@@ -397,7 +397,14 @@ func (k Keeper) ChanUpgradeOpen(
 		panic(fmt.Sprintf("counterparty channel state should be in one of [%s, %s, %s]; got %s", types.TRYUPGRADE, types.ACKUPGRADE, types.OPEN, counterpartyChannelState))
 	}
 
-	err = k.connectionKeeper.VerifyChannelState(ctx, connection, proofHeight, proofChannel, portID, channelID, counterpartyChannel)
+	err = k.connectionKeeper.VerifyChannelState(
+		ctx,
+		connection,
+		proofHeight, proofCounterpartyChannel,
+		channel.Counterparty.PortId,
+		channel.Counterparty.ChannelId,
+		counterpartyChannel,
+	)
 	if err != nil {
 		return errorsmod.Wrapf(err, "failed to verify counterparty channel, expected counterparty channel state: %s", counterpartyChannel.String())
 	}
```

Comment:
nit: should we consolidate the error check to within the scope of the func call like it is in `startFlushUpgrade`

```go
if err := k.connectionKeeper.VerifyChannelState(...); err != nil {
    return errorsmod.Wrapf(err, "...")
}
```

Reply:
ah yea, definitely.